radmon/verf_radtime.fd/bad_chan.f90

```diff
@@ -0,0 +1,94 @@
+!$$$ subprogram documentation block
+!               .      .    .
+! subprogram:	bad_chan	build bad_chan.txt file
+!     prgmmr:   safford			date:  2009-11
+!
+! abstract:	This module contains code to build the bad_chan.txt file.
+!		The bad_chan.txt file reports the satellite and channel for
+!		which zero obs were found on a channel that is set for use
+!               and normally has data.
+!
+! program history log:
+!	2009-12-22 safford - initial coding
+!
+! contains:
+!
+!
+! attributes:
+!   language:  f90
+!   machine:   ibm RS/6000 SP
+!
+!$$$
+
+module bad_chan
+
+  implicit none
+  
+  private
+
+  public :: open_bad_chan_file
+  public :: write_bad_chan
+  public :: close_bad_chan_file
+
+  integer, parameter       :: funit = 14
+
+  contains
+
+
+    !-------------------------------------------------------------
+    !  create the bad_chan file
+    !-------------------------------------------------------------
+
+    subroutine open_bad_chan_file( date, cycle, fios )
+
+      !--- interface
+      character(8), intent( in )	:: date
+      character(8), intent( in )	:: cycle
+      integer, intent( out )		:: fios     
+    
+      !--- variables
+      logical 				:: lexist = .FALSE. 
+      character(60)                     :: fname
+
+
+!      write(*,*) '--> open_bad_chan_file, date, cycle = ', date, cycle
```

Comment:
This looks like commented code leftover from debugging.  Perhaps it should be removed?